src/library_memfs.js

```diff
@@ -7,7 +7,7 @@
 addToLibrary({
   $MEMFS__deps: ['$FS', '$mmapAlloc'],
 #if !ASSERTIONS
-  $MEMFS__postset: `
```

Comment:
Can you explain how `MEMFS.doesNotExistError` could get used before the postset run?  IIUC the `post` set code is emitted at the top level at the end of the library top level code.    Is the problem that other `postset` code wants to run first?

Wouldn't `preset` attempt to set `MEMFS.doesNotExistError` before `MEMFS` itself is created?

Reply:
Actually `__preset` just doesn't exist.  Only `__postset`.  So I'm not sure how this could work.